test_util/aws.py

```diff
@@ -55,7 +55,7 @@ def create_stack(self, name, template_url, user_parameters, deploy_timeout=60):
         log.info('Requesting AWS CloudFormation...')
         cf_parameters = []
         for k, v in user_parameters.items():
-            cf_parameters.append({'ParameterKey': k, 'ParameterValue': v})
+            cf_parameters.append({'ParameterKey': k, 'ParameterValue': str(v)})
```

Comment:
Can we take:

```yaml
parameters:
 - ParameterKey: foo
   ParameterValue: bar
 - ParameterKey: baz
   ParameterValue: bang
````

instead. I'm fine with doing the string conversion, but really want this to be direct pass-through to AWS / none of us manipulating. The str() effectively just ends up being "yaml types aren't what we want".